app/models/service_template.rb

```diff
@@ -56,6 +56,9 @@ class ServiceTemplate < ApplicationRecord
   has_many   :service_templates, :through => :service_resources, :source => :resource, :source_type => 'ServiceTemplate'
   has_many   :services
 
+  has_many :service_template_tenants
+  has_many :additonal_tenants, :through => :service_template_tenants
```

Comment:
@gtanzillo @jrafanie 
I guess that this PR will be primarily about naming this association